includes/controllers/class-user.php

```diff
@@ -988,6 +988,15 @@ public function get_user_view_title() {
 			[
 				'username' => hivepress()->request->get_param( 'username' ),
 			]
+		)->set_args(
```

Comment:
Please check this below when the user object is already available, it's better than making the query more complex. It's faster and the error is explicit, while with the query you can't be sure that the user exists, but he's not activated - it will also return no results if there's no user with this username.